schema/identity-attribute.json

```diff
@@ -1,5 +1,20 @@
 {
-  "$schema": "http://json-schema.org/draft-04/schema",
-  "allOf": [{ "$ref": "http://json-schema.org/draft-04/schema" }],
-  "required": ["id", "title", "description"]
+	"$schema": "http://json-schema.org/draft-07/schema",
+	"$id": "http://platform.selfkey.org/schema/attribute/identity-attribute.json",
+	"allOf": [
+		{ "$ref": "http://json-schema.org/draft-07/schema" },
+		{
+			"properties": {
+				"idAttribute": {
```

Comment:
I really don't like the `identityAttribute` > `idAttribute` abbreviation.

It leaves a lot of ambiguities and opportunities for misunderstanding when talking about our work.

Depending on context, saying ID attribute could mean any of:

- The `_id` field on a MongoDB document,
- the `id` attribute on a JavaScript/JSON object, or
- one of these identity attribute schemas.

There might even be some that I haven't thought of.

Keeping the name the full `identityAttribute` removes any chance of misunderstanding, and all it costs is 6 characters in a name which can be autocompleted by a capable editor anyway.

Reply:
You are correct. But I will be removing this field anyway

Reply:
fixed to identityAttribute